lua/lspconfig/server_configurations/biome.lua

```diff
@@ -0,0 +1,35 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    cmd = { 'rome', 'lsp-proxy' },
```

Comment:
Should it not be `biome` instead of `rome` here?

Reply:
oops, yes. Fixed.